### PR TITLE
Update Indonesian translation

### PR DESCRIPTION
**Pull request by CAT-ID**
Update Indonesian translation

### CODE REVIEW

File: resources/lang/ind.ini [hunk 1]
```diff
@@ -1,46 +1,112 @@
...
+flags.hollowclosed = "Berongga dengan ujung tertutup"
+flags.natural = "Alami"
+; tools
+; wand tool
+tool.wand = "Wand"
+tool.wand.lore.1 = "Klik kiri sebuah blok untuk mengatur posisi 1 dari suatu pilihan"
+tool.wand.lore.2 = "Klik kanan sebuah blok untuk mengatur posisi 2 dari suatu pilihan"
+tool.wand.lore.3 = "Gunakan // togglewand untuk mengganti fungsionalitasnya"
+tool.wand.disabled = "Alat tongkat dinonaktifkan. Gunakan // togglewand untuk mengaktifkannya kembali"
+tool.wand.setenabled = "Alat tongkat sekarang {% }!"
```
Comment (inxomnyaa) [inline resources/lang/ind.ini]: Please make sure to keep the number - it is a placeholder for "enabled/disabled"

File: resources/lang/ind.ini [hunk 2]
```diff
@@ -1,46 +1,112 @@
...
-ui.brush.select.type.clipboard = "Papan klip"
-; ui brush settings
-ui.brush.settings.title = "{%0} pengaturan kuas"
-; ui brush options
+ui.brush.content = "Menu utama sikat"
+ui.brush.create = "Buat baru"
+ui.brush.getsession = "Dapatkan sikat sesi"
+ui.brush.edithand = "Edit kuas di tangan"
+; brush settings
+ui.brush.settings.title = "{% 0} pengaturan kuas"
```
Comment (inxomnyaa) [inline resources/lang/ind.ini]: Same here, please remove the space

File: resources/lang/ind.ini [hunk 3]
```diff
@@ -1,46 +1,112 @@
...
+tool.wand.lore.2 = "Klik kanan sebuah blok untuk mengatur posisi 2 dari suatu pilihan"
+tool.wand.lore.3 = "Gunakan // togglewand untuk mengganti fungsionalitasnya"
+tool.wand.disabled = "Alat tongkat dinonaktifkan. Gunakan // togglewand untuk mengaktifkannya kembali"
+tool.wand.setenabled = "Alat tongkat sekarang {% }!"
+; debug tool
+tool.debug = "Alat Debug"
+tool.debug.lore.1 = "Klik kiri satu blok untuk mendapatkan informasi"
+tool.debug.lore.2 = "seperti nama dan nilai kerusakan suatu blok"
+tool.debug.lore.3 = "Gunakan // toggledebug untuk mengaktifkan fungsinya"
+tool.debug.disabled = "Alat debug dinonaktifkan. Gunakan // toggledebug untuk mengaktifkannya kembali"
```
Comment (inxomnyaa) [inline resources/lang/ind.ini]: Please remove the space before "toggledebug"

**Review by inxomnyaa — CHANGES_REQUESTED** [review]
Please fix the marked issues (probably a google translate thing - happened to me too)

**Review by CAT-ID — COMMENTED** [review]
fixed

**Review by CAT-ID — COMMENTED** [review]
done.

**Review by inxomnyaa — APPROVED** [review]

**Comment by inxomnyaa** [comment]
Many thanks!

**Pull request merged**